terragrunt/modules/wiz/main.tf

```diff
@@ -1,6 +1,205 @@
+          "health:DescribeAffectedEntities",
+          "health:DescribeAffectedEntitiesForOrganization",
+          "health:DescribeEntityAggregates",
+          "health:DescribeEventAggregates",
+          "health:DescribeEventDetails",
+          "health:DescribeEventDetailsForOrganization",
+          "health:DescribeEventTypes",
+          "health:DescribeEvents",
+          "health:DescribeEventsForOrganization",
+          "health:DescribeHealthServiceStatusForOrganization",
+          "kafka:ListClusters",
+          "kendra:DescribeDataSource",
+          "kendra:DescribeIndex",
+          "kendra:ListDataSources",
+          "kendra:ListIndices",
+          "kendra:ListTagsForResource",
+          "kinesisanalytics:ListApplications",
+          "kinesisanalytics:DescribeApplication",
+          "kinesisanalytics:ListTagsForResource",
+          "kinesisvideo:ListStreams",
+          "kinesisvideo:ListTagsForStream",
+          "kinesisvideo:GetDataEndpoint",
+          "kms:GetKeyRotationStatus",
+          "kms:ListResourceTags",
+          "lambda:GetFunction",
+          "lambda:GetLayerVersion",
+          "logs:ListTagsForResource",
+          "profile:GetDomain",
+          "profile:ListDomains",
+          "profile:ListIntegrations",
+          "s3:GetBucketNotification",
+          "s3:GetMultiRegionAccessPointPolicy",
+          "s3:ListMultiRegionAccessPoints",
+          "ses:DescribeActiveReceiptRuleSet",
+          "ses:GetAccount",
+          "ses:GetConfigurationSet",
+          "ses:GetConfigurationSetEventDestinations",
+          "ses:GetDedicatedIps",
+          "ses:GetEmailIdentity",
+          "ses:ListConfigurationSets",
+          "ses:ListDedicatedIpPools",
+          "ses:ListReceiptFilters",
+          "voiceid:DescribeDomain",
+          "wafv2:GetLoggingConfiguration",
+          "wafv2:GetWebACLForResource",
+          "wisdom:GetAssistant",
+          "macie2:ListFindings",
+          "macie2:GetFindings",
+          "identitystore:List*",
+          "identitystore:Describe*",
+          "sso-directory:Describe*",
+          "sso-directory:ListMembersInGroup",
+          "cloudwatch:GetMetricStatistics"
+        ],
+        "Effect" : "Allow",
+        "Resource" : "*"
+      },
+      {
+        "Action" : [
+          "ec2:CopySnapshot",
+          "ec2:CreateSnapshot",
+          "kms:CreateKey",
+          "kms:DescribeKey",
+          "ec2:GetEbsEncryptionByDefault",
+          "ec2:DescribeSnapshots"
+        ],
+        "Effect" : "Allow",
+        "Resource" : "*"
+      },
+      {
+        "Action" : [
+          "ec2:CreateTags"
+        ],
+        "Effect" : "Allow",
+        "Resource" : "arn:aws:ec2:*::snapshot/*"
+      },
+      {
+        "Action" : "kms:CreateAlias",
+        "Effect" : "Allow",
+        "Resource" : [
+          "arn:aws:kms:*:*:alias/wizKey",
+          "arn:aws:kms:*:*:key/*"
+        ]
+      },
+      {
+        "Action" : [
+          "kms:CreateGrant",
```

Comment:
This seems like it allows Wiz to effectively bypass any KMS policies we might establish? We don't today, but if we e.g. managed signing keys as a kms resource I think this would allow wiz to sign things?

Reply:
Since we don't use KMS right now, lets just drop these permissions. We can revisit explicit allowances if we ever are worried about vulnerabilities in that aspect (Hint: I don't think we ever will be)